src/sequence-diagram.js

```diff
@@ -77,6 +86,27 @@
 	}
 
 /******************
+ * Drawing-related extra diagram methods.
+ ******************/
+
+	// These functions return the x-offset from the lifeline centre given the current Execution nesting-level.
+	function executionMarginLeft(level) {
+		if (level < 0) {
```

Comment:
A level < 0 indicates that there are no executions (just the actor/lifeline) It is impossible to go lower than -1, the parser will throw an error.